website/src/form-context.tsx

```diff
@@ -35,15 +41,30 @@ export const FormProvider = (props: { children: any }) => {
       }
     },
     onSubmit: (values) => {
-      setIsEditing(false)
-
-      runUpdate({
-        word: {
-          id: values.word["id"],
-          source: values.word["source"],
-          commentary: values.word["commentary"],
-        },
-      })
+      if (cherokeeRepresentation === Dailp.CherokeeOrthography.Taoc) {
+        setIsEditing(false)
+
+        // Create an array of MorphemeSegmentUpdate type to send to the backend.
+        const updatedSegments: Array<Dailp.MorphemeSegmentUpdate> = values.word[
+          "segments"
+        ].map((segment) => ({
+          system: Dailp.CherokeeOrthography.Taoc,
```

Comment:
Shouldn't we be able to supply this value from the preferences?

Reply:
I assumed this since only TAOC is supported for updating segments right now, but I can go ahead and change this to use the preferences!